assets/pages/oneoff-contributions/components/oneoffContributionsPayment.jsx

```diff
@@ -6,24 +6,26 @@ import React from 'react';
 import { connect } from 'react-redux';
 import type { Dispatch } from 'redux';
 
-import StripePopUpButton from 'components/paymentButtons/stripePopUpButton/stripePopUpButton';
```

Comment:
Can `stripePopUpButton.jsx` be deleted?

Reply:
Deleted here: 683a1ee. Good spot @tsop14